test/components.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow } from 'enzyme';
+import { Link } from 'react-router';
+
+import AppHeader from '../src/components/AppHeader';
+import CurrentTweet from '../src/components/CurrentTweet';
+import Tweet from '../src/components/Tweet';
+import CountryList from '../src/components/CountryList';
+import Flag from '../src/components/Flag';
+
+describe('<AppHeader />', () => {
+  it('should contain link to / and /feed', () => {
+    const wrapper = shallow(<AppHeader />);
+
+    const links = wrapper.find(Link);
+
+    expect(links.find({ to: '/' }).length).toBe(1);
+    expect(links.find({ to: '/feed' }).length).toBe(1);
+  });
```

Comment:
Jeg er enig i at de lenkene trengs for at appen skal være komplett. Men da mener jeg at man det er bedre med en mer overordnet test. denne blit veldig implementasjonsavhengig? Men det er kanskje bra?